packages/owl-bot/src/owl-bot.ts

```diff
@@ -290,7 +290,7 @@ export async function handlePullRequestLabeled(
       defaultBranch: payload?.repository?.default_branch,
     },
     octokit,
-    false
+    isBotAccount(payload.sender.login) ? true : false
```

Comment:
Do you need this ternary?

Reply:
```suggestion
    isBotAccount(payload.sender.login)
```

Reply:
No, left over from original logic I was writing.